codecov.yml

```diff
@@ -4,6 +4,3 @@ coverage:
       default:
         target: auto
         threshold: 2%
-ignore:
-  - "test_*.py"
-  - "*/**/tests/*"
```

Comment:
@kaxil why did you remove this? We don't want test files counting in code coverage otherwise we'll get incorrect reports that coverage is going down when we write tests. Is there somewhere else where this is being handled?